src/copy-filter.js

```diff
@@ -8,6 +8,8 @@ const prune = require('./prune')
 const targets = require('./targets')
 
 const DEFAULT_IGNORES = [
+  '(/|\b)package-lock\\.json$',
+  '(/|\b)yarn\\.lock$',
```

Comment:
Hmm. I just saw the test failure. I think this is a better pattern:
```suggestion
  '/package-lock\\.json$',
  '/yarn\\.lock$',
```

Reply:
Yeah - this should cover any lock files in any directory.